downstream/modules/platform/ref-controller-configure-host-name-notifications.adoc

```diff
@@ -2,9 +2,9 @@
 
 = Configuring the `controllerhost` hostname for notifications
 
-In xref:controller-configure-system[System settings], you can replace `https://controller.example.com` in the *Base URL of The Controller Host* field with your preferred hostname to change the notification hostname.
+From the *System settings* page, you can replace `https://controller.example.com` in the *Base URL of the Service* field with your preferred hostname to change the notification hostname.
```

Comment:
```suggestion
From the *System settings* page, you can replace `\https://controller.example.com` in the *Base URL of the Service* field with your preferred hostname to change the notification hostname.
```
Add a backslash in front of the URL to prevent it rendering as a clickable link in the doc.